azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatServiceListener.kt

```diff
@@ -138,11 +151,29 @@ internal class ChatServiceListener(
                         dispatch(ParticipantAction.ParticipantsAdded(participants = it.infoModel.participants))
                     }
                     ChatEventType.PARTICIPANTS_REMOVED -> {
-                        dispatch(ParticipantAction.ParticipantsRemoved(participants = it.infoModel.participants))
+
+                        dispatch(
+                            ParticipantAction.ParticipantsRemoved(
+                                participants = it.infoModel.participants,
+                                localParticipantRemoved = isLocalParticipantRemoved(it.infoModel.participants, localParticipantInfoModel)
```

Comment:
There is a chance collection of it.infoModel.participants contains multiple users, including local.
We should check if it contains local and post for local participant removed: ParticipantAction.ParticipantsRemoved
Plus to check if there is any user who is not local and post another ParticipantAction.ParticipantsRemoved

Reply:
isLocalParticipantRemoved does iterate over all participants and compare to local to set true/false.

updated isLocalParticipantRemoved to a one liner though.